source/qb64pe.bas

```diff
@@ -4475,6 +4471,21 @@ DO
                                         sfheader = 1
                                         GOTO GotHeader
                                     END IF
+                                    ' a740g: Fallback to source path
+                                    libpath$ = FixDirectoryName(idepath$) + og_libpath$
```

Comment:
Yes. You are correct, I actually did not think about that at all. It would be nice to have it relative to the `$INCLUDE` file because `$INCLUDE` itself works that way. I'll take a look at it. Would you know where in the source the `$INCLUDE` path resolution stuff is?

Reply:
I've made the changes so that it now looks for C++ header files relative to the `$INCLUDE` file.

Attaching my test code here so that we remember to add it to our tests suite once you fix `compile_tests.sh`.
[header_lib_test.zip](https://github.com/QB64-Phoenix-Edition/QB64pe/files/11057475/header_lib_test.zip)

Reply:
Added issue #324 for updating `compile_tests.sh`.